scipy/optimize/_dual_annealing.py

```diff
@@ -158,14 +157,14 @@ def __init__(self, lower, upper, callback=None):
         self.upper = upper
         self.callback = callback
 
-    def reset(self, func_wrapper, rand_state, x0=None):
+    def reset(self, func_wrapper, rand_gen, x0=None):
         """
         Initialize current location is the search domain. If `x0` is not
         provided, a random location within the bounds is generated.
         """
         if x0 is None:
-            self.current_location = self.lower + rand_state.random_sample(
-                len(self.lower)) * (self.upper - self.lower)
+            self.current_location = rand_gen.uniform(self.lower, self.upper,
+                                                     size=len(self.lower))
```

Comment:
similar comment on whitespace

Reply:
It's PEP8 compliant for line length (and other stuff)